app/controllers/todos_controller.rb

```diff
@@ -1,17 +1,18 @@
 class TodosController < ApplicationController
+  before_filter :require_user
```

Comment:
I would recommend to use `before_action` - thats a new standard/way in Rails. Does the same job as `before_filter` though.